Translations/fr.po

```diff
@@ -9135,12 +9140,11 @@ msgid ""
 "man could possess. I do not know what secrets it holds, my friend, but "
 "finding this stone would certainly prove most valuable."
 msgstr ""
-"Griswold parle de la Roche Céleste qui était destinée à l'enclave située à "
-"l'est. Elle y était emmené pour une étude plus approfondie. Cette pierre "
-"brillait d'une énergie qui accordait en quelque sorte une vision plus grande "
-"qu'un homme normal ne pouvait posséder. Je ne sais pas quels secrets elle "
-"recèle, mon ami, mais trouver cette pierre serait certainement très "
-"intéressant."
+"Griswold parle de la Roche Céleste qui était destinée à l'enclave d'Orient. "
+"Elle y était emmenée pour une étude plus approfondie. Cette pierre brillait "
```

Comment:
```suggestion
"Elle y a été emmenée pour une étude plus approfondie. Cette pierre brillait "
```

Reply:
The problem here is that the stone didn’t get to the destination, asthe caravan was attacked when the stone **was being carried**. That’s what I intended to say with the imperfect.